src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs

```diff
@@ -204,6 +204,25 @@ public UserTypeRecordsFormatDefinition()
             }
         }
         #endregion
+        #region User Types - Record structs
```

Comment:
:memo: Verified this showed up:
![image](https://user-images.githubusercontent.com/12466233/113897463-bcda5c80-977f-11eb-835d-b7aa3dedc0a2.png)